### PR TITLE
Reorg registration

### PR DESCRIPTION
**Pull request by Loquacity**
~**Merge https://github.com/uyuni-project/uyuni-docs/pull/573 first!!**~

# Description

Reorganises the client registration section.

TO COME IN A DIFFERENT PR: 
* Add new contact method section for Salt Minion

# Target branches

Which product version does this PR apply to?

- [ ] Manager-4.0
- [ ] Manager-4.1
- [x] Manager-4.2
- [x] Uyuni

# Links

Partial https://github.com/SUSE/spacewalk/issues/12477

### CODE REVIEW

**Comment by Loquacity** [comment]
image

File: modules/client-configuration/nav-client-configuration-guide.adoc [hunk 1]
```diff
@@ -37,9 +37,15 @@ endif::[]
...
 ** xref:registration-overview.adoc[Client Registration]
-*** xref:registration-webui.adoc[Register with the Web UI]
-*** xref:registration-bootstrap.adoc[Register with Bootstrap]
-*** xref:registration-cli.adoc[Register on the Command Line]
+*** xref:registration-methods.adoc[Registration Methods]
+**** xref:registration-webui.adoc[Register with the Web UI]
+**** xref:registration-bootstrap.adoc[Register with Bootstrap]
+**** xref:registration-cli.adoc[Register on the Command Line]
+// Registering Clients on a Proxy
+*** xref:client-proxy.adoc[Client Registration on a Proxy]
```
Comment (keichwa) [inline modules/client-configuration/nav-client-configuration-guide.adoc:45]: registration *on* / register *to*  --is this intentionally?

File: modules/client-configuration/pages/registration-methods.adoc [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+There are several ways to register clients to your {productname} Server.
+
+* For Salt clients, we recommend that you register clients using the {productname} {webui}.
+    For more information, see xref:client-configuration:registration-webui.adoc[].
+* If you want more control over the process, have to register many clients, or are registering traditional clients, we recommend that you create a bootstrap script.
+    For more information, see xref:client-configuration:registration-bootstrap.adoc[].
+* For Salt clients and even more control over the process, executing single commands on the command line can be useful.
+    For more information, see xref:client-configuration:registration-cli.adoc[].
+
+The client must have the date and time synchronized correctly with the {productname} server before registration.
```
Comment (keichwa) [inline modules/client-configuration/pages/registration-methods.adoc:13]:
```suggestion
The client must have the date and time synchronized correctly with the {productname} Server before registration.
```

**Review by keichwa — APPROVED** [review]

File: modules/client-configuration/nav-client-configuration-guide.adoc (hunk 1 above)
Comment (Loquacity) [reply modules/client-configuration/nav-client-configuration-guide.adoc:45]: Good point, it should be `to`.

**Review by keichwa — APPROVED** [review]

**Pull request merged**